common/src/traits.rs

```diff
@@ -59,7 +59,7 @@ impl<'a> TestFunctionExt for &'a str {
     }
 
     fn is_test(&self) -> bool {
-        self.starts_with("test")
+        self.starts_with("test") || self.is_invariant_test()
```

Comment:
this could lead to some side effects because the test runner differentiates between test and invariant test.